presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/credential/CredentialProviderType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc.credential;
+
+public enum CredentialProviderType
+{
+    inline,
+    file
```

Comment:
format this as:
```
public enum CredentialProviderType
{
    INLINE,
    FILE,
    /**/;
}
```

That way adding new enum will modify only a single line. Better git history.